src/script/page/RightSidebar/ConversationDetails/components/ConversationDetailsHeader/ConversationDetailsHeader.tsx

```diff
@@ -120,7 +125,14 @@ const ConversationDetailsHeader: FC<ConversationDetailsHeaderProps> = ({
             >
               {displayName && <span className="conversation-details__name">{displayName}</span>}
 
-              {canRenameGroup && <Icon.Edit className="conversation-details__name__edit-icon" />}
+              {canRenameGroup && (
```

Comment:
Remove title from the div otherwise screenreader is repeating the same thing twice

 <div
              className="conversation-details__name"
              title={t('tooltipConversationDetailsRename')}> Remove this since you have added this to the button which is correct